django/db/models/fields/json.py

```diff
@@ -156,7 +167,7 @@ def as_sql(self, compiler, connection):
         return "JSON_CONTAINS(%s, %s)" % (lhs, rhs), params
 
 
-class ContainedBy(PostgresOperatorLookup):
+class ContainedBy(FieldGetDbPrepValueMixin, PostgresOperatorLookup):
```

Comment:
These two lookups have a `rhs` that require a `JSONField` value thus inheriting from `FieldGetDbPrepValueMixin` makes sense.